src/Microsoft.ML.Transforms/GroupTransform.cs

```diff
@@ -180,35 +180,42 @@ public override RowCursor[] GetRowCursorSet(Func<int, bool> predicate, int n, Ra
         }
 
         /// <summary>
+        /// The output schema collects columns used to group columns (aka group-key columns) columns being grouped from the source scheam.
```

Comment:
>The output schema collects [](start = 12, length = 26)

Output schema doesn't do anything right, it just contains information. To claim it collects something is wrong. You can state what it's created by some actions, but I don't think it's right to say it's doing actions by itself.

Reply:
Ok.

---
In reply to: [245809414](https://github.com/dotnet/machinelearning/pull/2060#discussion_r245809414) [](ancestors = 245809414)